docs/install.md

```diff
@@ -14,147 +14,194 @@ See the License for the specific language governing permissions and
+docker.  
+This installation method uses `--triton-launch-mode=local` by
+default, as all the dependencies will be available.<br><br>
+
+**1. Clone Model Analyzer's Git Repository**
 
-$ docker build -t model-analyzer --build-arg TRITONSDK_BASE_IMAGE=<name of the built SDK image> .
+```
+git clone https://github.com/triton-inference-server/model_analyzer.git -b <rXX.YY>
 ```
 
-## Triton SDK Container
+**Replacing** `<rXX.YY>` with the version of Model Analyzer you want to install<br><br>
 
-You can also use the Triton SDK docker
-container available on the [NVIDIA GPU Cloud
-Catalog](https://ngc.nvidia.com/catalog/containers/nvidia:tritonserver). You can
-pull and run the SDK container with the following commands:
+**2. Build the Docker Image**
 
 ```
-$ docker pull nvcr.io/nvidia/tritonserver:22.05-py3-sdk
+cd ./model_analyzer
+
+docker build --pull -t model-analyzer .
 ```
 
-If you are not planning to run Model Analyzer with
-`--triton-launch-mode=docker`, You can run the SDK container with the following
-command: 
+Model Analyzer's Dockerfile bases the container on the latest `tritonserver`
+containers from NGC.<br><br>
+
+**3. Run the Container**
 
 ```
-$ docker run -it --gpus all --net=host nvcr.io/nvidia/tritonserver:22.05-py3-sdk
+docker run -it --rm --gpus all \
+    -v /var/run/docker.sock:/var/run/docker.sock \
+    -v <path-to-triton-model-repo>:<path-to-triton-model-repo> \
+    -v <path-to-output-model-repo>:<path-to-output-model-repo> \
+    --net=host model-analyzer
 ```
 
-You will need to build and install the Triton server binary inside the SDK
-container if you want to use the local mode. See the Triton [Installation
-docs](https://github.com/triton-inference-server/server/blob/main/docs/build.md)
-for more details. 
+**Replacing** `<path-to-triton-model-repo>` with the
+**_absolute_ _path_** to the directory where the `Triton` model repository
+will be located.  
+**Replacing** `<path-to-output-model-repo>` with the
+**_absolute_ _path_** to the directory where the `output` model repository
+will be located.<br><br>
 
-If you intend to use `--triton-launch-mode=docker`, which is recommended with 
-this method of using Model Analyzer when using the SDK container,
-you will need to mount the following: 
-   * `-v /var/run/docker.sock:/var/run/docker.sock` allows running docker
-      containers as sibling containers from inside the Triton SDK container.
-      Model Analyzer will require this if run  with
-      `--triton-launch-mode=docker`.
-   * `-v <path-to-output-model-repo>:<path-to-output-model-repo>` The
-      ***absolute*** path to the directory where the output model repository
-      will be located (i.e. parent directory of the output model repository).
-      This is so that the launched Triton container has access to the model
-      config variants that Model Analyzer creates.
+## Main with Local Launch Mode
+
+---
+
+This method allows you build the `main` branch of Model Analyzer.  
```

Comment:
This method allows you to build a docker container using the `main` branch of Model Analyzer's repository.